sphere-convenience/src/main/java/io/sphere/sdk/client/metrics/ObservedDuration.java

```diff
@@ -0,0 +1,35 @@
+package io.sphere.sdk.client.metrics;
+
+import io.sphere.sdk.client.SphereRequest;
+
+/**
+ * Contains duration information for one aspect for one execution of a {@link SimpleMetricsSphereClient#execute(SphereRequest)}.
+ *
+ * @see SimpleMetricsSphereClient
+ */
+public interface ObservedDuration {
+    /**
+     * The duration in milliseconds.
+     * @return duration
+     */
+    Long getDurationInMilliseconds();
+
+    /**
+     * The {@link SimpleMetricsSphereClient} adds a request ID to each {@link SphereRequest} so that matching {@link ObservedDuration}s can be matched.
```

Comment:
It is not clear to me what you mean, but I'll try rephrasing it:
"..., so that ObservedDurations belonging to the same request can be associated."?